src/analyze/analysis_handler.go

```diff
@@ -0,0 +1,60 @@
+package analyze
+
+import (
+	"BGPAlert/blt_mad"
+	"BGPAlert/common"
+	"BGPAlert/shake_alert"
+	"fmt"
+	"sort"
+	"time"
+)
+
+// Reads in a windowChannel for Window objects, parses the objects, and then calls the specified analysis functions
+func AnalyzeBGPMessages(windowChannel chan common.Window) {
+	for window := range windowChannel {
+		fmt.Println("Received Window: ")
+		bucketMap := window.BucketMap
+
+		// Convert BucketMap to a map of timestamp to length of messages
+		lengthMap := make(map[time.Time]float64)
+
+		for timestamp, messages := range bucketMap {
+			lengthMap[timestamp] = float64(len(messages))
+		}
+
+		// Turn map into sorted array of frequencies by timestamp
+		sortedFrequencies := getSortedFrequencies(lengthMap)
+
+		fmt.Println(sortedFrequencies)
+
+		fmt.Println("BLT MAD Outliers: ")
```

Comment:
Nit Pick: If you want to print things on a single line you can use fmt.Printf("TEXT HERE %+v", variable)

Reply:
And as you probs know you can use the '\n' to put things on a new line if thats what you wanted